pkg/dao/repository_config.go

```diff
@@ -489,3 +481,70 @@ func (r repositoryConfigDaoImpl) ValidateUrl(orgId string, url string, response
 	}
 	return nil
 }
+
+func (r repositoryConfigDaoImpl) ValidateUrlPresence(url string, response *api.RepositoryValidationResponse) *string {
```

Comment:
Would it be more precise to call this `ValidateMetadataPresence`? Or is it also doing something else?

Reply:
i think that makes sense, renamed

Reply:
also renamed these to start with lower case to indicate private methods (not declared on the interface)